wordpress-mobile/WordPress-iOS
# Issues/9185 prime only wpcom users

Refs #9185 

This is a small change to the alert for Prime for Push to only show it when the user has logged into a wpcom account.

To test:
- Start with a fresh install 
- Login to a self-hosted site
- Ensure that the push notification primer _doesn't_ appear (as in #9215). *This is the major change of this PR*
- Login to a wpcom account
- Ensure the primer _does_ appear

A gift for you, @ScoutHarris!
